.eslintrc-tsc.js

```diff
@@ -0,0 +1,13 @@
+const config = require('./.eslintrc.js');
```

Comment:
We need a separate config for typescript linting as of eslint 8, as eslint now parser typescript but often does it poorly.